plugins/cpu/fu-cpu-device.c

```diff
@@ -35,6 +35,64 @@ fu_cpu_device_to_string (FuDevice *device, guint idt, GString *str)
 				    fu_cpu_device_has_flag (self, FU_CPU_DEVICE_FLAG_SMAP));
 }
 
+static const gchar *
+fu_cpu_device_convert_vendor (const gchar *vendor)
```

Comment:
Were these sourced from kernel?  Might be good to reference in a comment where to get updates from

Reply:
Wikipedia! Is that good enough?

Reply:
Probably..